sql/besieged_mirrors.sql

```diff
@@ -0,0 +1,67 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `besieged_mirrors`
+--
+
+DROP TABLE IF EXISTS `besieged_mirrors`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `besieged_mirrors` (
+  `mobid` int(11) unsigned NOT NULL,
+  `destroyed` tinyint(1) NOT NULL,
+  `stronghold_id` tinyint(2) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`mobid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `besieged_mirrors`
+--
+
+LOCK TABLES `besieged_mirrors` WRITE;
+/*!40000 ALTER TABLE `besieged_mirrors` DISABLE KEYS */;
+INSERT INTO `besieged_mirrors` VALUES (16998838,1,3);
```

Comment:
Do these _have_ to be hard coded IDs? Could these be done another way?

Reply:
I think the way to look up mirrors that are still not destroyed would be less elegant. Currently we can perform a query to obtain the mirror count. 
I suppose you could look at mirrors per "region", with mobs spawning in areas of a specific region. Youd still want to track if a specific mobid is broken or not, so it needs to be tracked somewhere.

Reply:
ID shifts, is the thing I'm trying to avoid here. Since these are all named the same, and there are 24 total (3x sequential groups of 8) it's very easy to do a runtime query by name, or something involving offsets.

Reply:
Better yet, we can see in `mob_spawn_points` that there are 8 per zone, and they exist in blocks. You can key on zoneid+index, and look up the actual IDs at runtime.

Reply:
Ah gotcha. I can change the structure for that.

Reply:
Done. Changed to mirror_index which are 0 indexed. We can add these to the base ids.